docs/source/changelog.md

```diff
@@ -12,6 +12,239 @@ and as we merge [breaking changes in pull
 requests](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pulls?q=is%3Apr+is%3Aclosed+label%3Abreaking),
 this list should be updated.
 
+## 2.0
+
+### 2.0.0
+
+#### Highlights
+
+- **TODO**
+
+  TODO
+
+#### Security critical upgrade notice
+
+This is only relevant if you have configured `...networkPolicy.egress` for
+`hub`, `proxy.chp`, `proxy.traefik`, or `singleuser`.
+
+If you have configured `...networkPolicy.egress` for `hub`, `proxy.chp`,
+`proxy.traefik` or `singleuser` to restrict the permissions to establish
+outbound network connections, this upgrade is likely to escalate those
+permissions unless you revise your configuration. The new configuration
+`...networkPolicy.egressAllowRules` are by default granting most of the egress
+permissions previously granted by default via the `...networkPolicy.egress`
+configuration, and `...networkPolicy.egress` are now by default not providing
+any permissions.
+
+If you for example had overridden the previously very permissive default value
+of `singleuser.networkPolicy.egress` to be a bit less permissive, you should if
+you are unsure disable all `singleuser.networkPolicy.egressAllowRules` like this
+to not risk escalating the permissions.
+
+```yaml
+singleuser:
+  networkPolicy:
+    egressAllowRules:
+      cloudMetadataServer: false
+      dnsPortsPrivateIPs: false
+      nonPrivateIPs: false
+      privateIPs: false
+```
+
+For more details, see the documentation on [Kubernetes Network
+Policies](https://zero-to-jupyterhub.readthedocs.io/en/latest/administrator/security.html#kubernetes-network-policies)
+and the configuration reference entries under
+[`...networkPolicy.egress`](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#hub-networkpolicy-egress)
+and
+[`...networkPolicy.egressAllowRules`](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#hub-networkpolicy-egressallowrules).
+
+#### Upgrade instructions
+
+1. Adjust your configuration based on the security notice above if it applies to
+   your deployment.
+2. Prepare a user image with a modern version of JupyterHub installed. Install
+   `jupyterhub>=2.3.1` via pip or `jupyterhub-base>=2.3.1` via conda/mamba in
+   the image. Configure to use this new image, which is typically done via
+   `singleuser.image` of as part of `singleuser.profileList`.
+3. Read and adjust your configuration to the breaking changes below.
+
+#### Breaking changes
+
+- The following PRs have breaking labels since 1.1.3 release (1.1.4 and 1.2.0 are backport prs)
+
+  - use image for singleuser.cmd by default - https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2449
```

Comment:
````suggestion

##### Default to using the container image's command instead of `jupyterhub-singleuser` https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2449
Z2JH now launches the container's default command (equivalent to setting `CMD` in a `Dockerfile`) instead of overriding it. To restore the old bvehaviour set
```yaml
singleuser:
  cmd: jupyterhub-singleuser
```
````
I think it's cleaner to break these into separate subheadings instead of putting them all in a list, since they may contain a lot of context, but feel free to keep them as a list if you prefer.